core/src/test/java/org/pentaho/metaverse/analyzer/kettle/extensionpoints/job/JobLineageHolderMapTest.java

```diff
@@ -95,8 +95,6 @@ public void setUp() throws Exception {
             Collections.synchronizedMap( new MapMaker().weakKeys().makeMap() ) );
     Whitebox.setInternalState( TransLineageHolderMap.getInstance(), "lineageHolderMap",
             Collections.synchronizedMap( new MapMaker().weakKeys().makeMap() ) );
-    Whitebox.setInternalState( ExternalResourceCache.getInstance(), "transMap", new ConcurrentHashMap() );
```

Comment:
Not sure why an external test class is accessing internal state and verifying it of ExternalResourceCache. They should only use public methods.
Removing fragile and wrong methodology for unit tests.